userView/service/UpdatePassword.js

```diff
@@ -0,0 +1,22 @@
+const User = require('../models/User');
+const { StatusCodes } = require('http-status-codes');
+const { BadRequestError, NotFoundError, UnAuthenticatedError } = require('../errors/index');
+
+const updateUserPassword = async (req, res) => {
+    const { oldPassword, newPassword } = req.body;
+    if (!oldPassword || !newPassword) {
+      throw new BadRequestError('Please provide both values');
+    }
+    const user = await User.findOne({ _id: req.user.userId });
```

Comment:
User repository is expected to be called here too...Consistency is needed in your code